### ISSUE TITLE
mobx-react/native is not traspiled since 4.3.2

### ISSUE THREAD

**Issue by KsiBart** [mobxjs/mobx-react#428]
Hi, since `4.3.2` there is a problem with module `'mobx-react/native'`.

It seams it's not transpiled and while importing `import { observer, inject } from 'mobx-react/native'` It returns error:
```
/Users/me/.../node_modules/mobx-react/native.js:1
({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import {
                                                                                         ^^^^^^
    
SyntaxError: Unexpected token import
```

**Comment by mweststrate** [comment]
@KsiBart could you verify whether `mobx-react@4.4.2-fix-428` fixes the issue?

**Comment by mweststrate** [comment]
Released as 4.4.3, given the issue history, this probably / hopefully fixes it. Reopen if this isn't the case
